src/test/kotlin/ar/edu/unahur/obj2/semillasAlViento/SemillasTest.kt

```diff
@@ -0,0 +1,131 @@
+package ar.edu.unahur.obj2.semillasAlViento
+
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.shouldBe
+
+class SemillasTest : DescribeSpec({
+
+    var parcela = Parcela(10, 10, 8)
+    var parcelaAncha = Parcela(10, 5, 5)
+    var parcelaLarga = Parcela(5, 10, 15)
+
+    var menta = Menta(2019, 0.5f)//explicitamente decir que es float. Si no deberia ser double y seria lo mismo?
+    var menta2 = Menta(2019, 0.3f)//explicitamente decir que es float. Si no deberia ser double y seria lo mismo?
+    var menta3 = Menta(2019, 0.23f)//explicitamente decir que es float. Si no deberia ser double y seria lo mismo?
+
+    describe("Parcela") {
+
+
+        it("Calcular superficie") {
+            parcela.superficie().shouldBe(100)
+        }
+
+        it("Cantidad maxima de plantas cuando la parcela es mas ancha que larga") {
+            parcelaAncha.cantidadMaximaPlantas().shouldBe(10)
+        }
+
+        it("Cantidad maxima de plantas cuando la parcela es mas larga que ancha") {
+            parcelaLarga.cantidadMaximaPlantas().shouldBe(26)
+        }
+
+        it("Plantar en parcela") {
+            parcela.plantar(menta)
+            //Se cambiar el atributo cantidadPlantas por la funcion cantidadDePlantas()
+            parcela.cantidadDePlantas().shouldBe(1)
+        }
+
+        //Este requerimiento no lo tiene la parcela. La resposabilida se la dio a la planta
+        it("Tiene complicaciones") {
+            parcela.parcelaTieneComplicaciones()
```

Comment:
Este test está mal hecho, porque no tiene ningún `should...` y por ende no está probando nada. En este caso, deberían haber hecho un test para el caso true y otro para el false.